### ISSUE TITLE
Run htmlhint in force mode?

### ISSUE THREAD

**Issue by te-gpm** [bezoerb/grunt-php2html#6]
Hi,
thanks for your work, really love your plugin and have been using it in all of my latest projects!
However there is one small issue that keeps bugging me: I'm still trying to run the htmlhint part in force mode. 
Because many of my php files rely on the include function, the line numbers printed out in the console while running the task are off by several hundred lines. I could compare the line number in the resulting html file, but because htmlhint throws an error the file never gets created in my tmp folder for me to inspect. 
So basically I'm missing the files that I require the most. I got around the issue by running the same task without htmlhint before the main task, but that is both ugly and inefficient.

Any suggestions on how to fix this? 
Much appreciated! :)
